recipes/pygridtools/meta.yaml

```diff
@@ -0,0 +1,43 @@
+package:
+    name: pygridtools
+    version: 0.2
+
+source:
+    git_url: https://github.com/Geosyntec/pygridtools.git
+    git_tag: v0.2
```

Comment:
> Not sure how conda clones the repository (what is the depth of the clone?).

Default is to clone the whole history. There was an option to do shallow clones that was proposed. Should be in `conda build` 1.19.0 and later. Just set `git_depth` to the number of commits you would want.

> Users can delete them and re-add a new one with the same tag but fixing a problem, and then you have a checksum issue on your hands.

User could delete tags too. Generally, I feel these are bad practices, but you're right. We can always by bitten by them if someone does them anyways.

> PyPI solved that by forcing people to bump the version because PyPI does not accept the same file name/version for the same release even if you delete it.

Though that can get a bit extreme. Like if the upload got corrupted or PyPI had issues. I remember that happening to NumPy before.

<br>
In any event, given this is a relatively small repo. I am ok with it as is. If you want to try a shallow clone, feel free. If you guys work out something so it is on PyPI, I am good with that, as well. I leave it up to you.